### DISCUSSION TITLE
CPython 3.11 not production ready?

### DISCUSSION THREAD

**Discussion by hofrob** [celery/celery#8315] [General]
According to "What’s new in Celery 5.3 Emerald Rush", CPython 3.11 is not ready for production. Not sure if I'm missing something, so I'm creating a discussion first.

> This version is officially supported on CPython 3.8, 3.9 & 3.10 and is also supported on PyPy3.8+.

https://docs.celeryq.dev/en/stable/whatsnew-5.3.html#whatsnew-5-3

> ## Supported Python Versions
> 
> The supported Python versions are:
> 
> - CPython 3.8
> - CPython 3.9
> - CPython 3.10
> - PyPy3.8 7.3.11 (pypy3)
> 
> ## Experimental support
> 
> Celery supports these Python versions provisionally as they are **not production ready yet**:
> 
> - CPython 3.11

https://docs.celeryq.dev/en/stable/whatsnew-5.3.html#important-notes

Is this a mistake? We are on [3.11.3](https://peps.python.org/pep-0664/#bugfix-releases) now

**Comment by auvipy** [comment]
celery was not properly tested so it was mentioned. But it is tested in main branch now. should be released very soon.
